docs-compose/overwatch_docs/content/DeployOverwatch/UCEDeployment/CloudStorageAccessRequirements/AWS.md

```diff
@@ -3,9 +3,12 @@ title: "AWS"
 date: 2023-04-18T11:28:39-05:00
 weight: 1
 ---
-**Under Construction** -- we will be improving these docs shortly
 
-### AWS IAM/Policy required to set up for Storage Credentials
+### Creating AWS IAM Role/Policy for Storage Credentials
```

Comment:
At the very top here please provide me a bulleted list of the steps and have each bullet link to that section. I want to see an overview of what all needs to be completed.

Also, Try to make the headers more concise (below are just examples) but we want very simple headers with the description of what they do underneath. Do this for GCP as well.

* IAM Role Required for External Location Access
  * detail the requirements / need / reason for this requirement
* Add Trust Relationship for Storage IAM 
  * detail the requirements / need / reason for this requirement
* Provision Overwatch Access To Output Target
  * detail the requirements / need / reason for this requirement

Reply:
done